bot.py

```diff
@@ -146,12 +165,9 @@ def next_in_queue(self, _):
             loop = self.loop
 
             if self.voice_client.is_playing():
-                # þetta er hakk lausn, frekar ættum við að setja klasa inn sem after sem er callable, og þá getum við breytt
-                # hvað gerist þegar kallað er á after, þannig þegar stop trigger-ast þá getum við sleppt því að gera það sem við gerum
-                # venjulega
-                self.voice_client.pause()
+                self._stop()
```

Comment:
If there are 3 songs in the queue and someone calls !skip, block_after is set to True here so when the after_callback is called it will not autoplay the 2nd song in the queue, it will just stop right?

Reply:
Nevermind, I misread the code.